uktrade/export-wins-ui-mi
# Accessibility fixes

As part of the ongoing Data Hub accessibility work we have identified some accessibility fixes relating to this project, which are (mostly) fixed in this PR. The fixes are as follows:

- An aria-label has been added for the financial year dropdown on the main page
- The same dropdown has also been wrapped in a Fieldset
- The problem of duplicate links displaying on the homepage has been _partially_ fixed. The fix I have implemented will only detect duplicates with the same name, there are several Global HVCs that have been recorded incorrectly - this will need to be fixed in the database. There are currently 15 Global HVCs listed on prod but all of these link to the same three pages, this fix should bring that number down to eight.
- Updating the local header to include accessibility and security fixes

I also removed the 'Dependencies' badge in the Readme as it was connected with a defunct service.

### Test instructions:
For the first two fixes - Run the project in Google Chrome with the [Wave extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) installed and active. You should see the following:
<img width="321" alt="Screenshot 2020-09-09 at 16 53 34" src="https://user-images.githubusercontent.com/36161814/92622710-447ced00-f2bd-11ea-80ae-a01ffea312e9.png">

For the duplicate link fix, setup the project with fake stubs (`node src/data/create-backend-fake-stubs.js`). Navigate to the file `src > data > fake-stubs > backend > global_hvcs > index.2017.json` and add some duplicate data in `results`. Startup the project (with `npm run watch-fake-stub`) and these duplicate values should not appear.
